README.md

```diff
@@ -1,4 +1,6 @@
 # hello-world
 
 This is my trial repository.
```

Comment:
And this is my first comment through the Pull Requests tab, then the Files Changed tab